### ISSUE TITLE
The automatically checked off Walkthrough steps in vscode aren't read by NVDA screen reader and hence not accessible

### ISSUE THREAD

**Issue by melisaAkel** [melisaAkel/vscode#410]
Original issue: https://github.com/microsoft/vscode/issues/247108

The issue is that the information is not exposed to the user to allow for screen readers to announce events that are completed conditionally in a walkthrough step. A blind user would not know that a walkthrough step has been checked off.

I have attached a video of the automatically ticked off walkthrough steps on window reload.

- VS Code Version: 1.99.1
- OS Version: Microsoft Windows 11 Enterprise 10.0.26100 N/A Build 26100
